static/app/views/ddm/metricsExplorer.tsx

```diff
@@ -1,579 +1,6 @@
-                label: op,
-                value: op,
-              })) ?? []
-            }
-            disabled={!metricsQuery.mri}
-            value={metricsQuery.op}
-            onChange={option =>
-              updateQuery(router, {
-                op: option.value,
-              })
-            }
-          />
-          <CompactSelect
-            multiple
-            triggerProps={{prefix: t('Group by'), size: 'sm'}}
-            options={tags.map(tag => ({
-              label: tag.key,
-              value: tag.key,
-            }))}
-            disabled={!metricsQuery.mri}
-            value={metricsQuery.groupBy}
-            onChange={options =>
-              updateQuery(router, {
-                groupBy: options.map(o => o.value),
-                focusedSeries: undefined,
-              })
-            }
-          />
-        </PageFilterBar>
-      </QueryBuilderRow>
-      <QueryBuilderRow>
-        <MetricSearchBar
-          tags={tags}
-          mri={metricsQuery.mri}
-          disabled={!metricsQuery.mri}
-          onChange={query => updateQuery(router, {query})}
-          query={metricsQuery.query}
-        />
-      </QueryBuilderRow>
-    </QueryBuilderWrapper>
-  );
-}
-
-type MetricSearchBarProps = {
-  mri: string;
-  onChange: (value: string) => void;
-  tags: MetricsTag[];
-  disabled?: boolean;
-  query?: string;
-};
-
-function MetricSearchBar({tags, mri, disabled, onChange, query}: MetricSearchBarProps) {
-  const org = useOrganization();
-  const api = useApi();
-  const {selection} = usePageFilters();
-
-  const supportedTags: TagCollection = useMemo(
-    () => tags.reduce((acc, tag) => ({...acc, [tag.key]: tag}), {}),
-    [tags]
-  );
-
-  // TODO(ogi) try to use useApiQuery here
-  const getTagValues = useCallback(
-    async tag => {
-      const tagsValues = await api.requestPromise(
-        `/organizations/${org.slug}/metrics/tags/${tag.key}/`,
-        {
-          query: {
-            metric: mri,
-            useCase: getUseCaseFromMri(mri),
-            project: selection.projects,
-          },
-        }
-      );
-
-      return tagsValues.filter(tv => tv.value !== '').map(tv => tv.value);
-    },
-    [api, mri, org.slug, selection.projects]
-  );
-
-  const handleChange = useCallback(
-    (value: string, {validSearch} = {validSearch: true}) => {
-      if (validSearch) {
-        onChange(value);
-      }
-    },
-    [onChange]
-  );
-
-  return (
-    <WideSearchBar
-      disabled={disabled}
-      maxMenuHeight={220}
-      organization={org}
-      onGetTagValues={getTagValues}
-      supportedTags={supportedTags}
-      onClose={handleChange}
-      onSearch={handleChange}
-      placeholder={t('Filter by tags')}
-      defaultQuery={query}
-      savedSearchType={SavedSearchType.METRIC}
-    />
-  );
+// TODO(ddm): remove this after we have a dedicated metrics explorer for _admin
+export default function MetricsExplorer() {
```

Comment:
this will remove metrics expolorer in the admin temporarily, will be coordinated merge with a followup in getsentry